app/helpers/application_helper.rb

```diff
@@ -11,4 +11,13 @@ def path_to_current_user_profile
       edit_master_profile_path
     end
   end
+
+  def will_be_at(game)
+    case
+    when game.events.any? {|e| e.beginning_at.today? }
+      t('general.today')
+    else
+      t('general.tomorrow')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.